transaction/src/manifest/parser.rs

```diff
@@ -300,7 +300,8 @@ impl Parser {
             TokenKind::EddsaEd25519Signature |
             TokenKind::Decimal |
             TokenKind::PreciseDecimal |
-            TokenKind::NonFungibleId  => self.parse_scrypto_types(),
+            TokenKind::NonFungibleId |
+            TokenKind::Bytes => self.parse_scrypto_types(),
```

Comment:
Bytes are not "custom" scrypto type. Suggest independent method.

Reply:
Changed.